### PR TITLE
cedille: 1.0.0 -> 1.1.1

### PR DESCRIPTION
**Pull request by marsam**
###### Motivation for this change
Changelog: https://github.com/cedille/cedille/blob/master/CHANGELOG.md#cedille-11

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [X] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Review by tomberek — APPROVED** [review]
Thanks for cleaning up the expression! builds/runs on Ubuntu, did not test compiler output

**Comment by timokau** [comment]
Thanks, the cleanups look good!

For the future (not important enough to hold off merge here), its generally better to add yourself as a maintainer in a separate commit.

**Comment by timokau** [comment]
Aarch64 failure seems to be a faulty ofBorg instance.

**Pull request merged**
